flow/templates/expanse.sh

```diff
@@ -0,0 +1,54 @@
+{# Templated in accordance with: https://www.sdsc.edu/support/user_guides/expanse.html #}
+{% extends "slurm.sh" %}
+{% block tasks %}
+    {% set threshold = 0 if force else 0.9 %}
+    {% set cpu_tasks = operations|calc_tasks('np', parallel, force) %}
+    {% set gpu_tasks = operations|calc_tasks('ngpu', parallel, force) %}
+    {% if gpu_tasks %}
+        {% if not ('gpu' in partition or force) %}
+            {% raise "GPU operations require a GPU partition!" %}
+        {% endif %} {# End check for correct partition #}
```

Comment:
Doesn't seem necessary since the `if` is only two lines above.
```suggestion
        {% endif %}
```